ocs_ci/helpers/helpers.py

```diff
@@ -2790,3 +2791,42 @@ def fetch_used_size(cbp_name, exp_val=None):
             f"matching. Retrying"
         )
     return used_in_gb
+
+
+def check_pvs_present_for_ocs_expansion(sc="localblock"):
+    """
+    Check for pvs present for OCS cluster expansion
+
+    Args:
+        sc (str): Name of SC
+
+
+    Returns:
+        bool: True if PVS present, assert if PVS not present
```

Comment:
`assert` should be detailed under `Raises` docstring section

Reply:
Done